### PR TITLE
fix chainsaw 2h-ing

### LINKED ISSUE #17924: dual wielding chainsaws
Possible to dual wield chainsaws by having someone throw them and having the other person catching them.  This doesn't take the extra arm slot so you have a free hand and can repeat the process.

[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)

### PR DESCRIPTION
**Pull request by Iamgoofball**
fixes #17924

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -5,10 +5,13 @@
 				if(istype(AM, /obj/item))
 					var/obj/item/I = AM
 					if(isturf(I.loc))
-						put_in_active_hand(I)
-						visible_message("<span class='warning'>[src] catches [I]!</span>")
-						throw_mode_off()
-						return 1
+						if(istype(I, /obj/item/weapon/twohanded))
```
Comment (KorPhaeron) [inline code/modules/mob/living/carbon/carbon_defense.dm]: snowflake

Comment (KorPhaeron) [inline code/modules/mob/living/carbon/carbon_defense.dm]: yeah but if I tell you then you will steal those sweet tokens

Comment (Iamgoofball) [inline code/modules/mob/living/carbon/carbon_defense.dm]: exactly, so this PR stays as-is

Comment (Iamgoofball) [inline code/modules/mob/living/carbon/carbon_defense.dm]: also any other fixes would be a feature

**Comment by phil235** [comment]
Why are twohanded objects not unwielded when thrown?

**Comment by Thunder12345** [comment]
Won't this break catching anything that isn't two handed?

**Comment by Iamgoofball** [comment]
@phil235 they are, the problem is the chainsaw auto-wields when held

**Comment by PKPenguin321** [comment]
goof wait we should make it so if you try to catch an item with force/wielded force >= 15 and with sharpness it dismembers the arms/an arm instead
its funnier that way and should still solve the issue

**Comment by phil235** [comment]
>they are, the problem is the chainsaw auto-wields when held

Well there's the real issue: hacky chainsaw code

**Comment by Iamgoofball** [comment]
@phil235 it applies to all forced 2hand weapons, they do the same thing

**Comment by KorPhaeron** [comment]
Phil downvoted this and I trust phil with my life

**Comment by Iamgoofball** [comment]
fuck off

**Comment by Iamgoofball** [comment]
he downvoted it because of the name

**Pull request closed without merging**